core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -1227,7 +1230,7 @@ public void testCloseSharePartitionManager() throws Exception {
         Timer timer = Mockito.mock(SystemTimerReaper.class);
         Persister persister = Mockito.mock(Persister.class);
         SharePartitionManager sharePartitionManager = SharePartitionManagerBuilder.builder()
-                .withTimer(timer).withShareGroupPersister(persister).build();
+            .withTimer(timer).withShareGroupPersister(persister).build();
```

Comment:
nit: Was the indentation wrong earlier?

Reply:
I think it's just a difference of indentation settings. Also this piece of code resulted in a conflict for me, probably that's why the change. Didn't notice the change, I will revert this in the next commit, thanks